qemu/tests/cfg/dump_guest_core.cfg

```diff
@@ -14,7 +14,7 @@
     crash_cmd = "crash -i ${crash_script} /usr/lib/debug/lib/modules/%s/vmlinux ${vmcore_file}"
     dump_guest_memory_file = "/usr/share/qemu-kvm/dump-guest-memory.py"
     x86_64:
-        extra_params += " -device vmcoreinfo"
+        extra_params += " -device '{"driver":"vmcoreinfo"}'" 
```

Comment:
Recommend to use `vmcoreinfo = yes` to replace `extra_params += " -device '{"driver":"vmcoreinfo"}'" `